README.md

```diff
@@ -14,11 +14,26 @@ This plugin adds a new `$clipboard` magic property to all of your Alpine compone
 
 ## Installation
 
+### NPM
```

Comment:
Can we place the this section _after_ the CDN section please? I'd like to recommend the CDN installation first because that's the simplest and also the same as Alpine's own recommended method.

Reply:
Moved. 👍